cli/pkg/cli_utils/repeated_string_test.go

```diff
@@ -0,0 +1,139 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+package cli_utils
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+)
+
+func TestIsWellBehaved(t *testing.T) {
+	type testCase struct {
+		name string
+		str  string
+		res  bool
+	}
+
+	tcs := []testCase{
+		{
+			name: "okay string",
+			str:  "totally-okay",
+			res:  true,
+		},
+		{
+			name: "comma",
+			str:  "bad,comma",
+			res:  false,
+		},
+		{
+			name: "space",
+			str:  "bad space",
+			res:  false,
+		},
+		{
+			name: "dot",
+			str:  "something.yaml",
+			res:  true,
+		},
+		{
+			name: "slashes",
+			str:  "path/to/something",
+			res:  true,
+		},
+	}
+
+	for _, tc := range tcs {
+		tc := tc
+		t.Run(tc.name, func(t *testing.T) {
+			res := isWellBehavedString(tc.str)
+			if res != tc.res {
+				t.Fatalf("incorrect result for string %s, expected %v, got %v", tc.str, tc.res, res)
+			}
+		})
+	}
+}
+
+func TestParseCommandLineArgs(t *testing.T) {
+	type testCase struct {
+		name             string
+		argNames         []string
+		cmdArgs          []string
+		expectedValues   map[string]string
+		expectedErrorMsg string // should not be flakey
```

Comment:
please adapt to https://github.com/freiheit-com/kuberpult/blob/main/README_DEVELOPERS.md#always-use-cmpdiff
then we don't have to worry about flaky tests